src/app/replayer/replayer.ml

```diff
@@ -638,6 +638,8 @@ let write_replayer_checkpoint ~logger ~ledger ~last_global_slot_since_genesis
         [ ("max_canonical_slot", `String (Int64.to_string max_canonical_slot)) ] ;
     Deferred.unit )
 
+let mainnet_version = Mina_numbers.Txn_version.of_int 2
```

Comment:
Let’s call this migrating_from_version (or something? accepting suggestions) for two reasons: 1. soon berkeley will be running on mainnet 2. make it clear for when we want to do the next hardfork what the purpose of this version is